karaf/features/src/main/feature/feature.xml

```diff
@@ -291,40 +292,29 @@
     </feature>
 
     <feature name="brooklyn-container-service" version="${project.version}" description="Brooklyn Container Service and Location Targets">
-        <bundle start-level="85">mvn:org.apache.brooklyn/brooklyn-locations-container/${project.version}</bundle>
-
         <feature>brooklyn-core</feature>
-        
-<!-- the intention is to pull in these features to support deploy-to k8s 
-        <feature>kubernetes-client</feature>
-        <feature>openshift-client</feature>
-     however they pull in incompatible versions - jackson 2.7.5 and snakeyaml 1.17 -
-     so we declare instead the extra deps it declares, but skip those already pulled in.
-     these two we skip because we pull them in elsewhere (same versions, but don't redeclare fixed version)
-        <bundle dependency='true'>mvn:javax.validation/validation-api/1.1.0.Final</bundle>
-        <bundle dependency='true'>mvn:javax.servlet/javax.servlet-api/3.1.0</bundle>
-    these it declares at a different version, right version pulled in elsewhere
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/2.7.5</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/2.7.5</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/2.7.5</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.7.5</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/2.7.5</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/2.7.5</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/2.7.5</bundle>
-        <bundle dependency='true'>mvn:org.yaml/snakeyaml/1.17</bundle>
-    these are needed:  -->
-        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.generex/1.0.1_1</bundle>
+
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/${fasterxml.jackson.version}</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/${fasterxml.jackson.version}</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.datatype/jackson-datatype-jsr310/${fasterxml.jackson.version}</bundle>
+
+        <!-- (1) -->
```

Comment:
These numbers are cross-references to the readme, but that won't be obvious for someone looking at the source code - worth saying to look at the readme. Or move the comment/description into this file.